### ISSUE TITLE
TypeError: Cannot destructure property 'header' of 'handler.info' as it is undefined.

### ISSUE THREAD

**Issue by chrisl8888** [kettanaito/msw#1689]
### Prerequisites

- [X] I confirm my issue is not in the [opened issues](https://github.com/mswjs/msw/issues)
- [X] I confirm the [Frequently Asked Questions](https://mswjs.io/docs/faq) didn't contain the answer to my issue

### Environment check

- [X] I'm using the [latest](https://github.com/mswjs/msw/releases/latest) `msw` version
- [X] I'm using Node.js version 14 or higher

### Node.js version

v14.17.6

### Reproduction repository

http://www.github.com

### Reproduction steps

```javascript
import {setupServer} from 'msw/node';

const handlers = [...]
const server = setupServer(...handlers);
server.printHandlers();
```

Throws `TypeError: Cannot destructure property 'header' of 'handler.info' as it is undefined.`

Stacktrace from console

```
    at node_modules/msw/src/node/SetupServerApi.ts:145:15
          at Array.forEach (<anonymous>)
      at SetupServerApi.printHandlers (node_modules/msw/src/node/SetupServerApi.ts:144:14)
      at Object.<anonymous> (test/setup-after-env.js:15:12)
```

### Current behavior

using `msw@1.2.1`

### Expected behavior

Should not throw when calling printHandlers

**Comment by chrisl8888** [comment]
Resolved this; was passing an array of an array of handlers in my setup-after-env.js

```
const handlers => [
  <Function>,
  <Function>,
  <Array<Function>>
]
```


Would maybe recommend adding tests in `printHandlers.node.test.ts` for falsey handlers and skipping it and provide a warning?

https://github.com/mswjs/msw/blob/main/test/node/msw-api/setup-server/printHandlers.node.test.ts#L39

**Comment by kettanaito** [comment]
Hi. Thanks for reporting this. We should error on this at the setupServer level. We do have a check there but somehow it didn't trigger in your case. This issue is also not strictly related to the `.printHandlers()` method but to the call signature of `setupServer()`/`setupWorker()`. We need to improve runtime value guards there to prevent this kind of situations. 


Can be a good first issue to fix.

**Comment by zbyte64** [comment]
This is happening to me, ~~but I don't think I'm passing an array of arrays, just a flat list of RestHandlers~~ I had not destructured the array when calling server.use

**Comment by kettanaito** [comment]
@zbyte64, that's a good point. I don't think we have any warnings in the `.use()` methods 👍 Those should be added there too.

**Comment by kettanaito** [comment]
## Released: v2.0.9 🎉

This has been released in v2.0.9!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v2.0.9)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/2.0.9)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
